File: Beecrowd/2543 Jogatina UFPR/main.cpp
#include <iostream>
 
using namespace std;

int main() {
    int n, u, i, j, cont;

    while(cin >> n >> u) {
        cont = 0;
        for(int k = 0; k < n; k++){
            cin >> i >> j;
            if(i == u && j == 0){
                cont++;
            }
        }

        cout << cont << endl;
    }


    return 0;
}

File: Beecrowd/2543 Jogatina UFPR/main.py
while True:
    try:
        n, u = [int(_) for _ in input().split()]
        cont = 0
        for _ in range(n):

            i, j = [int(_) for _ in input().split()]
            if i == u and j == 0:
                cont += 1
        print(cont)
    except EOFError:
        break